pkgs/by-name/bo/bochs/package.nix

```
@@ -60,80 +67,86 @@ stdenv.mkDerivation (finalAttrs: {
-    "--enable-debugger" #conflicts with gdb-stub option
-    "--enable-debugger-gui"
-    "--enable-evex"
-    "--enable-fpu"
-    "--enable-gdb-stub=no" # conflicts with debugger option
-    "--enable-handlers-chaining"
-    "--enable-idle-hack"
-    "--enable-iodebug"
-    "--enable-large-ramfile"
-    "--enable-largefile"
-    "--enable-pci"
-    "--enable-repeat-speedups"
-    "--enable-show-ips"
-    "--enable-smp"
-    "--enable-vmx=2"
-    "--enable-svm"
-    "--enable-trace-linking"
-    "--enable-usb"
-    "--enable-usb-ehci"
-    "--enable-usb-ohci"
-    "--enable-usb-xhci"
-    "--enable-voodoo"
-    "--enable-x86-64"
-    "--enable-x86-debugger"
-  ] ++ lib.optionals enableSDL2 [
-    "--with-sdl2"
-  ] ++ lib.optionals enableTerm [
-    "--with-term"
-  ] ++ lib.optionals enableWx [
-    "--with-wx"
-  ] ++ lib.optionals enableX11 [
-    "--with-x"
-    "--with-x11"
-  ] ++ lib.optionals (!stdenv.isDarwin) [
-    "--enable-e1000"
-    "--enable-es1370"
-    "--enable-ne2000"
-    "--enable-plugins"
-    "--enable-pnic"
-    "--enable-sb16"
+    (lib.enableFeatureAs true "cpu-level" "6") # from 3 to 6
+    (lib.enableFeatureAs true "vmx" "2")
+
+    (lib.enableFeature false "gdb-stub") # conflicts with debugger option
+    (lib.enableFeature true "debugger") #conflicts with gdb-stub option
+
+    (lib.enableFeature true "a20-pin")
+    (lib.enableFeature true "avx")
+    (lib.enableFeature true "busmouse")
+    (lib.enableFeature true "cdrom")
+    (lib.enableFeature true "clgd54xx")
+    (lib.enableFeature true "configurable-msrs")
+    (lib.enableFeature true "debugger-gui")
+    (lib.enableFeature true "evex")
+    (lib.enableFeature true "fpu")
+    (lib.enableFeature true "handlers-chaining")
+    (lib.enableFeature true "idle-hack")
+    (lib.enableFeature true "iodebug")
+    (lib.enableFeature true "large-ramfile")
+    (lib.enableFeature true "largefile")
+    (lib.enableFeature true "pci")
+    (lib.enableFeature true "repeat-speedups")
+    (lib.enableFeature true "show-ips")
+    (lib.enableFeature true "smp")
+    (lib.enableFeature true "svm")
+    (lib.enableFeature true "trace-linking")
+    (lib.enableFeature true "usb")
+    (lib.enableFeature true "usb-ehci")
+    (lib.enableFeature true "usb-ohci")
+    (lib.enableFeature true "usb-xhci")
+    (lib.enableFeature true "voodoo")
+    (lib.enableFeature true "x86-64")
+    (lib.enableFeature true "x86-debugger")
+
+    (lib.withFeature enableSDL2 "sdl2")
+    (lib.withFeature enableTerm "term")
+    (lib.withFeature enableWx "wx")
+    (lib.withFeature enableX11 "x")
+    (lib.withFeature enableX11 "x11")
+
+    (lib.enableFeature (!stdenv.isDarwin) "e1000")
+    (lib.enableFeature (!stdenv.isDarwin) "es1370")
+    (lib.enableFeature (!stdenv.isDarwin) "ne2000")
+    (lib.enableFeature (!stdenv.isDarwin) "plugins")
+    (lib.enableFeature (!stdenv.isDarwin) "pnic")
+    (lib.enableFeature (!stdenv.isDarwin) "sb16")
   ];
 
+  outputs = [ "out" "lib" "doc" "man" ];
```

```suggestion
  outputs = [ "out" "doc" "man" ] ++ lib.optionals (!stdenv.isDarwin) [ "lib" ];
```

For unbreaking on aarch64-darwin.